stan/math/rev/fun/exp2.hpp

```diff
@@ -41,6 +41,15 @@ inline var exp2(const var& a) {
   });
 }
 
+template <typename T, require_eigen_t<T>* = nullptr>
+inline auto exp2(const var_value<T>& a) {
+  return make_callback_var(
+      a.val().unaryExpr([](auto&& x) { return std::exp2(x); }),
```

Comment:
Can probably just use the Stan Math `exp2`  here since that's vectorised as well